src/modules/dashboard/components/Dialogs/ControlSafeDialog/ControlSafeForm.tsx

```diff
@@ -260,6 +283,26 @@ const ControlSafeForm = ({
     [selectedContractMethods, handleSelectedContractMethods, setFieldValue],
   );
 
+  const handleTransactionTypeChange = (type: string, index: number) => {
+    const setContractFunction = (contractFunction: string) =>
+      setFieldValue(
+        `transactions.${index}.contractFunction`,
+        contractFunction,
+        true,
+      );
+    switch (type) {
+      case TransactionTypes.TRANSFER_FUNDS:
+        setContractFunction('transfer');
```

Comment:
Shouldn't these values be part of some enum/constant instead of a hardcoded string?